test/cartesian/ReferenceLine/ReferenceLine.spec.tsx

```diff
@@ -296,4 +296,62 @@ describe('<ReferenceLine />', () => {
       y2: -102.22222222222223,
     });
   });
+
+  test('throws when passed in invalid xAxisId', () => {
+    const viewBox: CartesianViewBox = { x: 1, y: 2 };
+    expect(() => {
+      render(
+        <BarChart
+          width={1100}
+          height={250}
+          barGap={2}
+          barSize={6}
```

Comment:
Can we further simplify this test by removing unneeded properties? These seem unrelated.

Reply:
Done